multichain-testing/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -101,41 +108,91 @@ test.after(async t => {
+  const instanceP: Promise<Instance> = vstorageClient
+    .queryData('published.agoricNames.instance')
+    .then(pairs => fromEntries(pairs)[contractName]);
+  const brandP: Promise<[string, Brand][]> = vstorageClient.queryData(
+    'published.agoricNames.brand',
   );
-  const instance = fromEntries(instances)[contractName];
 
-  // accept oracle operator invitations
-  await Promise.all(
-    oracleWds.map(makeDoOffer).map((doOffer, i) =>
-      doOffer({
-        id: toOracleOfferId(i),
-        invitationSpec: {
-          source: 'purse',
-          instance,
-          description: 'oracle operator invitation', // TODO export/import INVITATION_MAKERS_DESC
-        },
+  const description = 'oracle operator invitation';
+
+  const doOffer = makeDoOffer(wd);
+  const self = harden({
+    getName: () => name,
+    acceptInvitation: async () => {
+      const instance = await instanceP;
+      await doOffer({
+        id: toOracleOfferId(idx),
+        invitationSpec: { source: 'purse', instance, description },
         proposal: {},
-      }),
-    ),
-  );
+      });
 
-  for (const name of keys(oracleMnemonics)) {
-    const addr = wallets[name];
-    await t.notThrowsAsync(() =>
-      retryUntilCondition(
-        () => vstorageClient.queryData(`published.wallet.${addr}.current`),
-        ({ offerToUsedInvitation }) => {
-          return offerToUsedInvitation[0][0] === `${name}-accept`;
-        },
+      await retryUntilCondition(
+        () => self.checkInvitation(),
+        check => check.usedInvitation,
         `${name} invitation used`,
+      );
+    },
+    checkInvitation: async () => {
+      const { Invitation } = fromEntries(await brandP);
+      const { offerToUsedInvitation, purses }: CurrentWalletRecord =
+        await vstorageClient.queryData(`published.wallet.${address}.current`);
+      type InvitationDetails = { description: string }; // TODO: import from Zoe
+      const { value: details } = purses.find(p => p.brand === Invitation)!
+        .balance as Amount<'set', InvitationDetails>;
+      const invitation = details.find(x => x.description === description);
+      const usedInvitation = offerToUsedInvitation.some(
+        ([k, _v]) => k === `${name}-accept`,
+      );
+      return { invitation, usedInvitation };
+    },
+  });
+  return self;
+};
+
+test.serial('oracles accept', async t => {
+  const { wallets, oracleWds, retryUntilCondition, vstorageClient } = t.context;
+  const operators = await Promise.all(
+    keys(oracleMnemonics).map((_k, idx) =>
+      makeTxOracle(
+        idx,
+        oracleWds[idx],
+        wallets[idx],
+        vstorageClient,
+        retryUntilCondition,
       ),
-    );
+    ),
+  );
+
+  const checks = await Promise.all(
+    operators.map(async op => {
+      const { invitation, usedInvitation } = await op.checkInvitation();
+      t.log({ name: op.getName(), invitation, usedInvitation });
+      t.true(invitation || usedInvitation, 'has or accepted invitation');
```

Comment:
Shouldn't this `map` function return `usedInvitation` for the logic below to work? If it doesn't return anything then `checks.some(Boolean)` would always be false right?